### PR TITLE
Update Gmax Pokemon legality in Battle Stadium and VGC

### PR DESCRIPTION
**Pull request by KrisXV**
https://pastebin.com/SLwFPJgB

these are the new Gmax Pokemon legal in series 3 according to Pigeons.

### CODE REVIEW

File: config/formats.js [hunk 1]
```diff
@@ -198,8 +198,10 @@ let Formats = [
 		minSourceGen: 8,
 		onBegin() {
 			const allowedGigantamaxes = [
-				"Charizard-Gmax", "Butterfree-Gmax", "Pikachu-Gmax", "Meowth-Gmax", "Eevee-Gmax", "Snorlax-Gmax",
-				"Corviknight-Gmax", "Dreadnaw-Gmax", "Sandaconda-Gmax", "Centiskorch-Gmax",
+				"Alcremie-Gmax", "Appletun-Gmax", "Butterfree-Gmax", "Centiskorch-Gmax", "Charizard-Gmax","Coalossal-Gmax",
```
Comment (Zarel) [inline config/formats.js]:
```suggestion
				"Alcremie-Gmax", "Appletun-Gmax", "Butterfree-Gmax", "Centiskorch-Gmax", "Charizard-Gmax", "Coalossal-Gmax",
```

File: config/formats.js [hunk 2]
```diff
@@ -303,8 +303,10 @@ let Formats = [
...
```
Comment (Zarel) [inline config/formats.js]:
```suggestion
				"Alcremie-Gmax", "Appletun-Gmax", "Butterfree-Gmax", "Centiskorch-Gmax", "Charizard-Gmax", "Coalossal-Gmax",
```

**Comment by DaWoblefet** [comment]
We also have the Litten, Rowlet, Popplio, Bulbasaur, and Squirtle lines freed, in addition to Galarian Slowpoke and all previously banned Regional formes. 

**Do not implement this change until February 29th at the earliest**. VGC tournaments do not switch over until March 1st, and inconveniently, there is a Regional Championship that weekend (a major). Once the tournament starts on the 29th this is fine to go through.

**Comment by Zarel** [comment]
Why are we using completely different Gmax bans for VGC and Battle Stadium, anyway?

Can we just unify them all into the same system?

**Comment by KrisXV** [comment]
did @Zarel @DaWoblefet

File: config/formats.js [hunk 3]
```diff
@@ -195,6 +195,7 @@ let Formats = [
 			battle: 3,
 		},
 		ruleset: ['Standard GBU'],
+		banlist: ['Copperajah-Gmax', 'Duraludon-Gmax', 'Garbodor-Gmax', 'Gengar-Gmax', 'Machamp-Gmax'],
 		minSourceGen: 8,
 		onValidateSet(set) {
```
Comment (Zarel) [inline config/formats.js]: You still need to delete this, right?

File: config/formats.js [hunk 4]
```diff
@@ -300,6 +294,7 @@ let Formats = [
 			battle: 4,
 		},
 		ruleset: ['Standard GBU'],
+		banlist: ['Copperajah-Gmax', 'Duraludon-Gmax', 'Garbodor-Gmax', 'Gengar-Gmax', 'Machamp-Gmax'],
 		minSourceGen: 8,
 		onValidateSet(set) {
```
Comment (Zarel) [inline config/formats.js]: And this?

File: config/formats.js (hunk 3 above)
Comment (KrisXV) [reply config/formats.js]: i definitely deleted those idk why they are still there. weird

**Pull request merged**